engine/src/main/java/org/entando/entando/web/page/validator/PageValidator.java

```diff
@@ -125,7 +132,10 @@ public void validateGroups(String pageCode, PagePositionRequest pageRequest, Err
     public void validatePagesStatus(String pageCode, PagePositionRequest pageRequest, Errors errors) {
         IPage parent = this.getPageManager().getDraftPage(pageRequest.getParentCode()),
                 page = this.getPageManager().getDraftPage(pageCode);
+        logger.debug("page.isOnline() {}", page.isOnline());
```

Comment:
If we're going to log these we should include an id or something so we know which page we're logging about.